node/hack/governor/src/index.ts

```diff
@@ -136,8 +136,8 @@ axios
                   await (async () => {
                     const provider = new JsonRpcProvider(
                       new Connection({
-                        // fullnode: "https://fullnode.mainnet.sui.io",
-                        fullnode: "https://sui-mainnet.g.allthatnode.com/full/json_rpc",
+                         fullnode: "https://fullnode.mainnet.sui.io",
```

Comment:
Why is the URL being changed here?

Reply:
The script wasn't working with the other rpc. So, I tried this one in the comments and it worked. I can change it back though.

Reply:
What was the issue?

Reply:
The request was simply failing - like a bad rpc.

Reply:
That RPC has been spotty for a while, I've also had more luck with the fullnode endpoint so happy to switch that to the default again